src/gui_common/TweakedCheckBox.tscn

```diff
@@ -0,0 +1,35 @@
+[gd_scene load_steps=9 format=2]
+
+[ext_resource path="res://src/gui_common/fonts/Jura-DemiBold-Smaller.tres" type="DynamicFont" id=1]
```

Comment:
I much prefer Jura-DemiBold-AlmostSmaller.tres if you want a smaller font as this one is too small.